### DISCUSSION TITLE
Bring over save Preview Image from OpenLoco

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17691] [Feature requests] [answered]
OpenLoco has a fancy preview image for your worlds, small screenshot/paused render of the last viewport. Would definitely make it easier to overview your parks.

This feature is already present in RCTC.

![image](https://user-images.githubusercontent.com/1516893/182524328-a43ed619-5f16-462c-85f0-721d3b4e5538.png)

![image0](https://user-images.githubusercontent.com/1516893/182526261-6b8682de-a70f-42e1-88b9-b63134fc5ece.png)

As a bonus it could be a cool statistic to see how much irl game time you have in said park.

**Comment by ocalhoun6** [comment]
A similar preview could also be neat to have for scenarios when you're selecting which scenario to play.

**Reply by karst** [reply]
Could even make it so you can snap a specific preview image while building the scenario.

**Reply by duncanspumpkin** [reply]
In Locomotion the image is drawn when hitting save.

**Comment by 733737** [comment]
would it necessarily need to be 4:3? I would like an option for 16x9 preview

**Reply by karst** [reply]
4:3 seems alot cleaner than a 16:9 image for this specific usecase.

**Comment by J-PIE-314** [comment]
In PR #22646, @AaronVanGeffen is planning to implement a map preview.

**Reply by AaronVanGeffen** [reply]
That's for scenarios though, not save games. Though I'd like to implement this request as well.

**Comment by Gymnasiast** [comment, marked as answer]
Has now been implemented for both saves ans scenarios.
